packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.test.ts

```diff
@@ -998,11 +1060,12 @@ export function testSandwichComposing() {
 		);
 		it("[move, move, modify, move] ↷ [del]", () =>
 			withConfig(() => {
+				const nodeId: NodeId = { localId: brand(3) };
```

Comment:
3 is already use by mo3 and mi3
```suggestion
				const nodeId: NodeId = { localId: brand(4) };
```